app/views/navbar/_navbar.html.erb

```diff
@@ -1,4 +1,4 @@
-<% if user_signed_in? %>
+ <% if user_signed_in? %>
```

Comment:
- [ ] Your app is looking great, nice job! However, when you are in the expenses list, there is no link to go back to the categories list, it would be great to have some way to navigate back and forward:

<img width="379" alt="Screen Shot 2022-08-13 at 2 18 15 PM" src="https://user-images.githubusercontent.com/83717285/184507864-81a62c8b-e14f-4c68-b710-4b17629816c0.png">

Reply:
Thank you, I will correct it's